src/mapeo-manager.js

```diff
@@ -389,6 +393,16 @@ export class MapeoManager extends TypedEmitter {
     // TODO: Close the project instance instead of keeping it around
     this.#activeProjects.set(projectPublicId, project)
 
+    // load default config
+    // TODO: see how to expose warnings to frontend
+    /* eslint-disable no-unused-vars */
+    let warnings
+    if (this.#defaultConfigPath) {
+      warnings = await project.importConfig({
+        configPath: new URL(this.#defaultConfigPath, import.meta.url).pathname,
```

Comment:
I think `defaultConfigPath` should be an absolute path, no need to support relative paths here.